php/php.editor/test/unit/src/org/netbeans/modules/php/editor/actions/FixUsesPerformerTest.java

```diff
@@ -401,6 +401,24 @@ public void testGH5578_08() throws Exception {
         Options options = new Options(false, false, false, false, true);
         performTest("// test^", createSelections(selections, ItemVariant.Type.CLASS), true, options);
     }
+    
+    public void testMultipleDefaultNSUse_01() throws Exception {
+        String[] selections = new String[] {"\\NS2\\B"};
+        Options options = new Options(false, false, false, false, true);
+        performTest("// test^", createSelections(selections, ItemVariant.Type.CLASS), true, options);
+    }
+    
+    public void testMultipleDefaultNSUse_02() throws Exception {
+        String[] selections = new String[] {"\\NS2\\D"};
+        Options options = new Options(false, false, false, false, true);
+        performTest("// test^", createSelections(selections, ItemVariant.Type.CLASS), true, options);
+    }
+    
+    public void testWholeFileUse() throws Exception {
+        String[] selections = new String[] {"\\NS2\\B", "\\NS2\\D"};
+        Options options = new Options(false, false, false, false, true);
+        performTest("// test^", createSelections(selections, ItemVariant.Type.CLASS), true, options);
+    }
```

Comment:
Can you add a test case for trait?

Reply:
Done

Reply:
I meant the case that `use NS2\F;`(trait) is inserted. Sorry for not being clear...

Reply:
Hope I understood it right.